oml/utils/misc_torch.py

```diff
@@ -126,31 +124,33 @@ def _check_is_sequence(val: Any) -> bool:
         return False
 
 
-def drop_duplicates_by_ids(ids: List[Hashable], data: Tensor, sort: bool = True) -> Tuple[List[Hashable], Tensor]:
+TData = Tuple[List[Any], BoolTensor, FloatTensor, LongTensor, Tensor, np.ndarray]
```

Comment:
Redundant types?
~BoolTensor, FloatTensor, LongTensor~, Tensor
or
BoolTensor, FloatTensor, LongTensor, ~Tensor~

Reply:
Agree, prefer the first option